pages/task.tsx

```diff
@@ -4,13 +4,19 @@ import data from '../data/IT2810Høst2018.json';
 import Header from '../components/header';
 import { Button, Grid } from '@mui/material';
 import Link from 'next/link';
-import { getAssessments, noRemainingAnswers } from '../functions/helpFunctions';
+import {
+  getAssessments,
+  noRemainingAnswers,
+  isAllTasksAssessed,
+} from '../functions/helpFunctions';
 import { useEffect, useState } from 'react';
 
 const Task: NextPage = () => {
   const totalTasks: number =
     data.ext_inspera_candidates[0].result.ext_inspera_questions.length;
-  const taskNumbers: number[] = Array.from(Array(totalTasks).keys());
```

Comment:
This is also not directly related to this PR, I just caught som bugs because of it, and so I changed it here. Some changes (removing `+1 ` or `-1`) was made due to this, both here and in `helpFunctions.tsx`.